mergify_engine/subscription.py

```diff
@@ -115,22 +116,45 @@ def to_dict(self) -> SubscriptionDict:
             "features": [f.value for f in self.features],
         }
 
+    RETENTION_SECONDS = 60 * 60 * 24 * 3  # 3 days
+    VALIDITY_SECONDS = 3600
+
+    async def _have_expired(self) -> bool:
+        expire_at: int = await self.redis.ttl(
+            f"subscription-cache-owner-{self.owner_id}"
+        )
+        if expire_at < 0:
+            return True
+
+        return expire_at < (self.RETENTION_SECONDS - self.VALIDITY_SECONDS)
```

Comment:
I'd just use `self.VALIDITY_SECONDS` here, no?

Reply:
I try to rewrite the condition to make it clearer